nucliadb/src/nucliadb/writer/api/v1/export_import.py

```diff
@@ -75,6 +83,52 @@ async def start_kb_export_endpoint(request: Request, kbid: str):
         return CreateExportResponse(export_id=export_id)
 
 
+@only_for_onprem
+@api.post(
+    f"/{KBS_PREFIX}/import",
+    summary="Create a KB from an export and import its content",
+    tags=["Knowledge Boxes"],
+    # response_model=CreateImportResponse,
```

Comment:
can you add the param that excludes the endpoint def from the docs?

Reply:
Sure